src/components/apphost.js

```diff
@@ -1,447 +1,445 @@
-            features.push('externalpremium');
-        }
+    if (browser.operaTv || browser.tizen || browser.orsay || browser.web0s) {
+        features.push('exit');
+    } else {
+        features.push('exitmenu');
+        features.push('plugins');
+    }
 
-        if (!browser.operaTv) {
-            features.push('externallinkdisplay');
-        }
+    if (!browser.operaTv && !browser.tizen && !browser.orsay && !browser.web0s && !browser.ps4) {
+        features.push('externallinks');
+        features.push('externalpremium');
+    }
 
-        if (supportsVoiceInput()) {
-            features.push('voiceinput');
-        }
+    if (!browser.operaTv) {
+        features.push('externallinkdisplay');
+    }
 
-        if (supportsHtmlMediaAutoplay()) {
-            features.push('htmlaudioautoplay');
-            features.push('htmlvideoautoplay');
-        }
+    if (supportsVoiceInput()) {
+        features.push('voiceinput');
+    }
 
-        if (browser.edgeUwp) {
-            features.push('sync');
-        }
+    if (supportsHtmlMediaAutoplay()) {
+        features.push('htmlaudioautoplay');
+        features.push('htmlvideoautoplay');
+    }
 
-        if (supportsFullscreen()) {
-            features.push('fullscreenchange');
-        }
+    if (browser.edgeUwp) {
+        features.push('sync');
+    }
 
-        if (browser.tv || browser.xboxOne || browser.ps4 || browser.mobile) {
-            features.push('physicalvolumecontrol');
-        }
+    if (supportsFullscreen()) {
+        features.push('fullscreenchange');
+    }
 
-        if (!browser.tv && !browser.xboxOne && !browser.ps4) {
-            features.push('remotecontrol');
-        }
+    if (browser.tv || browser.xboxOne || browser.ps4 || browser.mobile) {
+        features.push('physicalvolumecontrol');
+    }
 
-        if (!browser.operaTv && !browser.tizen && !browser.orsay && !browser.web0s && !browser.edgeUwp) {
-            features.push('remotevideo');
-        }
+    if (!browser.tv && !browser.xboxOne && !browser.ps4) {
+        features.push('remotecontrol');
+    }
 
-        features.push('displaylanguage');
-        features.push('otherapppromotions');
-        features.push('displaymode');
-        features.push('targetblank');
-        features.push('screensaver');
+    if (!browser.operaTv && !browser.tizen && !browser.orsay && !browser.web0s && !browser.edgeUwp) {
+        features.push('remotevideo');
+    }
 
-        webSettings.getMultiServer().then(enabled => {
-            if (enabled) features.push('multiserver');
-        });
+    features.push('displaylanguage');
+    features.push('otherapppromotions');
+    features.push('displaymode');
+    features.push('targetblank');
+    features.push('screensaver');
 
-        if (!browser.orsay && (browser.firefox || browser.ps4 || browser.edge || supportsCue())) {
-            features.push('subtitleappearancesettings');
-        }
+    webSettings.enableMultiServer().then(enabled => {
```

Comment:
```suggestion
    webSettings.getMultiServer().then(enabled => {
```

renamed [here](https://github.com/jellyfin/jellyfin-web/commit/1fe25e14ddb6131be62b1ed4dbb12f3a821a5702#diff-caa5f991c737c4fe47a9dbc920955cdbR21-R27)